grpc/src/main/java/com/linecorp/armeria/server/grpc/FramedGrpcService.java

```diff
@@ -300,9 +300,6 @@ private <I, O> void startCall(ServerMethodDefinition<I, O> methodDef, ServiceReq
             call.setListener((Listener<I>) EMPTY_LISTENER);
             final Metadata metadata = new Metadata();
             call.close(GrpcStatus.fromThrowable(statusFunction, ctx, t, metadata), metadata);
```

Comment:
I'm wondering if we want to also pass the throwable so that the cause is exposed better.
What do you think other maintainers?

```suggestion
            call.close(t);
```

Reply:
Nice idea! Additionally, the call path would create metadata from `throwable` using `generateMetadataFromThrowable()`.

Reply:
I see. okay~